batchflow/tests/model_save_load_test.py

```diff
@@ -77,59 +74,85 @@ def test_run(self, save_path, pipelines, model_class):
 
         Predictions from save_pipeline and from load_pipeline should be equal
         """
-        save_pipeline, load_pipeline = pipelines(model_class)
-
         save_tmpl = (Pipeline()
                      .save_model('model', path=save_path + I("current").str())
-                     .train_model('model', B('images'), B('labels'), fetches='loss'))
-
-        save_pipeline = save_pipeline + save_tmpl
-        save_pipeline.run(BATCH_SIZE, n_epochs=1, bar=True)
-        saved_predictions = save_pipeline.get_variable('predictions')
+                    )
 
         load_tmpl = (Pipeline()
-                     .load_model('dynamic', C('model_class'), 'model', path=save_path + I("current").str()))
+                     .load_model('model', C('model_class'), 'dynamic', path=save_path + I("current").str())
+                    )
+
+        train_pipeline, predict_pipeline = pipelines(model_class)
```

Comment:
`train_pipeline` -> `train_predict_pipeline` ?

Reply:
I've change to `train_pipeline` and `inference_pipeline`